packages/react-components/react-infobutton/src/components/InfoButton/useInfoButtonStyles.ts

```diff
@@ -1,33 +1,161 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { createCustomFocusIndicatorStyle } from '@fluentui/react-tabster';
+import { iconFilledClassName, iconRegularClassName } from '@fluentui/react-icons';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { tokens } from '@fluentui/react-theme';
 import type { InfoButtonSlots, InfoButtonState } from './InfoButton.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const infoButtonClassNames: SlotClassNames<InfoButtonSlots> = {
   root: 'fui-InfoButton',
-  // TODO: add class names for all slots on InfoButtonSlots.
-  // Should be of the form `<slotName>: 'fui-InfoButton__<slotName>`
+  // this className won't be used, but it's needed to satisfy the type checker
+  popover: 'fui-InfoButton__popover',
+  content: 'fui-InfoButton__content',
 };
 
 /**
  * Styles for the root slot
  */
-const useStyles = makeStyles({
-  root: {
-    // TODO Add default styles for the root element
+const useButtonStyles = makeStyles({
+  base: {
+    alignItems: 'center',
+    backgroundColor: tokens.colorTransparentBackground,
+    ...shorthands.borderColor('transparent'),
```

Comment:
Maybe this? This sets the color to 'transparent' everywhere except the high contrast theme.

I'm not 100% this is the correct thing to do here but if it is there are several other places where a token could be swapped in for 'transparent'.

```suggestion
    ...shorthands.borderColor(tokens.colorTransparentStroke),
```